pants.ini

```diff
@@ -1,6 +1,6 @@
 [GLOBAL]
-pants_version:1.15.0.dev3
-plugins:[
+pants_version : 1.15.0.dev3
```

Comment:
Is this a related change? I've generally seen `key: value` , but presumably the matching code works for any combination.

Reply:
Yes, this unfortunately is the result of us using in `ci.py` `ConfigParser` to rewrite back our `pants.ini` after the tests. It doesn't allow us to specify the spacing beyond if we want no spaces or a space on both sides. 

Originally, our regex parsing wasn't flexible enough to allow a space before the `:` (even though it's perfectly valid `ini` syntax), so we had to use no spaces. Now that we can use either style, I changed back to the spaces because it's `ConfigParser`'s default and I think is easier to read.